src/data/constants/instructorInfo.js

```diff
@@ -0,0 +1,8 @@
+export const FAIL_INSTRUCTOR_INFO = 'FAIL_INSTRUCTOR_INFO';
+export const RECEIVE_INSTRUCTOR_INFO = 'RECEIVE_INSTRUCTOR_INFO';
+export const REQUEST_INSTRUCTOR_INFO = 'REQUEST_INSTRUCTOR_INFO';
+export const CREATE_INSTRUCTOR = 'CREATE_INSTRUCTOR';
+export const CREATE_INSTRUCTOR_SUCCESS = 'CREATE_INSTRUCTOR_SUCCESS';
+export const CREATE_INSTRUCTOR_FAIL = 'CREATE_INSTRUCTOR_FAIL';
+
+export const UUID_REGEX = /^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/;
```

Comment:
Probably shouldn't duplicate this specific constant. Either move it somewhere central or import it from the original place.

Reply:
`constants/utils.js` seems icky to me; any suggestions?